ci-operator/step-registry/openshift-qe/installer/gcp/ipi/ovn/arm/openshift-qe-installer-gcp-ipi-ovn-arm-workflow.yaml

```diff
@@ -0,0 +1,20 @@
+workflow:
+  as: openshift-qe-installer-gcp-ipi-ovn-arm
+  steps:
+    env: 
+      OCP_ARCH: arm64
+    pre:
+    - ref: gcp-provision-vpc
```

Comment:
@paigerube14 I think for a gcp ipi install without vpc, some steps in pre are not necessary. Like in cucushift-installer-rehearse-gcp-ipi-ovn-provision, there are only the following steps
  - chain: ipi-conf-gcp
  - ref: ovn-conf 
  - chain: ipi-install
  - ref: enable-qe-catalogsource
  - chain: cucushift-installer-check

Reply:
I copied the wrong steps, updated now thanks!